<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="distrFunc" native="true">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>10</y>
      <width>601</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="Params">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>731</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Параметры моделирования</string>
    </property>
    <widget class="QLineEdit" name="curSeed">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>50</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="isrdSeed">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>20</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Истинное случ. число</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="startButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Старт</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lambdaPar">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>20</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nPar">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>80</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>100</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="uniqValN">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>80</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>32768</string>
     </property>
    </widget>
    <widget class="QLabel" name="seedlb">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>50</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Seed:</string>
     </property>
    </widget>
    <widget class="QLabel" name="uniqValNlb">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>80</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Уник. зн.</string>
     </property>
    </widget>
    <widget class="QLabel" name="lambdaParlb">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>20</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Пар-р λ </string>
     </property>
    </widget>
    <widget class="QLabel" name="nParlb">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>80</y>
       <width>47</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Кол-во </string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>261</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Вариант 6:

Случайная величина η - время обслуживания 
покупателя в кассе магазина. 
Распределена экспоненциально с показателем λ.</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="rvTables">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>731</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Таблицы значений и характеристик случайной величины</string>
    </property>
    <widget class="QTableWidget" name="rvTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>711</width>
       <height>51</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string>X_i</string>
      </property>
     </row>
    </widget>
    <widget class="QTableWidget" name="rvStatsTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>711</width>
       <height>71</height>
      </rect>
     </property>
     <row>
      <property name="text">
       <string/>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Mη</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>X̄</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>|Mη - X̄|</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Dη</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>S^2</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>|Dη - S^2|</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Me^</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>R^</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>D </string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QTabWidget" name="other">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>1331</width>
      <height>412</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="histogramm">
     <attribute name="title">
      <string>Гистограмма</string>
     </attribute>
     <widget class="QCustomPlot" name="histGraph" native="true">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>0</y>
        <width>591</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="histStats">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>701</width>
        <height>121</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>z_j</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>fη(z_j)</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>n_j/(n*d_j)</string>
       </property>
      </row>
     </widget>
     <widget class="QLineEdit" name="histN">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QPushButton" name="histStart">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>141</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Построить гистограмму</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="histRanges">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>120</y>
        <width>531</width>
        <height>51</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>Границы</string>
       </property>
      </row>
     </widget>
     <widget class="QLabel" name="histNlb">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>131</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Число отр.</string>
      </property>
     </widget>
     <widget class="QLabel" name="maxinfo">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>42</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;max| n_j/(n*d_j) - fη(z_j) |&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:6pt;&quot;&gt;j = 1..k&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="max">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>50</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="hypothesis">
     <attribute name="title">
      <string>Проверка гипотезы</string>
     </attribute>
     <widget class="QPushButton" name="hypStart">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Проверить гипотезу H0:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="hypN">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>69</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLabel" name="hypNlb">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Число отр.</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="hypRanges">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>531</width>
        <height>51</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>Границы</string>
       </property>
      </row>
     </widget>
     <widget class="QTableWidget" name="hypH0">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>160</y>
        <width>531</width>
        <height>51</height>
       </rect>
      </property>
      <row>
       <property name="text">
        <string>q_j</string>
       </property>
      </row>
     </widget>
     <widget class="QLabel" name="HypSignlb">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>201</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Уров. знач.</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="hypSign">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>200</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>0.1</string>
      </property>
     </widget>
     <widget class="QLabel" name="hypReslb">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Результат:</string>
      </property>
     </widget>
     <widget class="QLabel" name="hypRes">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>280</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="hypTestlb">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>280</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>при |F(R0) = </string>
      </property>
     </widget>
     <widget class="QLabel" name="hypTest">
      <property name="geometry">
       <rect>
        <x>316</x>
        <y>280</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
